server/src/main/java/org/elasticsearch/index/mapper/InferenceFieldMapper.java

```diff
@@ -33,4 +33,14 @@ public interface InferenceFieldMapper {
      * @return The field's original value, or {@code null} if none was provided
      */
     Object getOriginalValue(Map<String, Object> sourceAsMap);
+
+    /**
+     * Get the field's embedding name
+     *
+     * @param fieldName The inference field name
+     * @return The field's full path to embedding value
+     */
+    static String getInferenceFieldName(String fieldName) {
+        return fieldName + ".inference.chunks.embeddings";
+    }
```

Comment:
This implementation should be provided by the concrete implementation of `InferenceFieldMapper`. Let's hold off on that though as it's not clear if we should filter just embeddings or the entire inference field.

Reply:
I wanted to use `SemanticTextField,` but it required me to add a dependency in the Java module `requires org. elasticsearch.inference`, and the build broke. I could not figure out how to get past that, so I went in a different direction.

Reply:
Yes, that's expected since `InferenceFieldMapper` is in `server` and `SemanticTextField` is in the inference plugin. In order for this to work, `SemanticTextFieldMapper` must implement this function as part of satisfying the interface.